hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/server/datanode/Replica.java

```diff
@@ -19,49 +19,56 @@
 
 import org.apache.hadoop.classification.InterfaceAudience;
 import org.apache.hadoop.hdfs.server.common.HdfsServerConstants.ReplicaState;
+import org.apache.hadoop.hdfs.server.datanode.fsdataset.FsVolumeSpi;
 
 /** 
  * This represents block replicas which are stored in DataNode.
  */
 @InterfaceAudience.Private
 public interface Replica {
   /** Get the block ID  */
-  public long getBlockId();
+  long getBlockId();
 
   /** Get the generation stamp */
-  public long getGenerationStamp();
+  long getGenerationStamp();
 
   /**
    * Get the replica state
    * @return the replica state
    */
-  public ReplicaState getState();
+  ReplicaState getState();
 
   /**
    * Get the number of bytes received
    * @return the number of bytes that have been received
    */
-  public long getNumBytes();
+  long getNumBytes();
   
   /**
    * Get the number of bytes that have written to disk
    * @return the number of bytes that have written to disk
    */
-  public long getBytesOnDisk();
+  long getBytesOnDisk();
 
   /**
    * Get the number of bytes that are visible to readers
    * @return the number of bytes that are visible to readers
    */
-  public long getVisibleLength();
+  long getVisibleLength();
```

Comment:
please do not change these interface methods. These changes are not required and makes backport harder.

Reply:
Thanks @jojochuang for your review. This change is to fix checkstyle. I will restore it.